src/modules/flow/http-server/http-server.c

```diff
@@ -313,43 +329,60 @@ sse_conn_closed(void *data, const struct sol_http_progressive_response *sse)
 }
 
 static int
+send_data(struct sol_http_progressive_response *client, struct sol_blob *blob)
+{
+    int r;
+
+    r = sol_http_progressive_response_feed(client, &sse_prefix);
+    SOL_INT_CHECK(r, < 0, r);
+
+    r = sol_http_progressive_response_feed(client, blob);
+    SOL_INT_CHECK(r, < 0, r);
+
+    r = sol_http_progressive_response_feed(client, &sse_suffix);
+    SOL_INT_CHECK(r, < 0, r);
+
```

Comment:
as someone said elsewhere, this function can be move to http server, something like `sol_http_progressive_response_feed_sse()`, then you don't need to replicate this here and in web-inspector.